### PR TITLE
add writers-program and cfp-program pages with revision

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: content/en/pages/writers-program/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+    - Best practices: Share tips and tricks to make the Developer Experience easier and more efficient.
+    
+    - Personal experiences: Share your own unique journey and lessons learned using mirrord in your software development cycle.
+    
+    - Feel free to suggest content related to mirrord and its surrounding technologies.
+- label: who?
+  title: Who can write for us?
+  content: >
+    We welcome applications from everyone, regardless of experience level. You must have some experience writing for a third party and are comfortable with receiving feedback on your writing.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:32]:
```suggestion
    We welcome applications from everyone, regardless of experience or role. All we ask is that you be comfortable with receiving feedback on your writing.
```

File: content/en/pages/writers-program/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+    - We do not publish posts that have appeared elsewhere.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:35]:
```suggestion
    - We do not publish posts that have already been published somewhere else.
```

File: content/en/pages/writers-program/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- label: why?
+  title: Why should you participate?
+  content: >
+    - Get paid: You can earn $150-$300 for each article you write that gets published.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:39]: Weird that this is the first time this is mentioned. Should probably be mentioned in the hero section. @Anita-ihuman @aviramha wdyt?

File: content/en/pages/writers-program/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    
+    - Help others: Your writing will help educate and empower other developers.
+    
+    - Build your reputation: Increase your visibility within global developer communities.
+
+    - Boost your portfolio: Showcase your writing skills and technical knowledge to potential employers.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:45]:
```suggestion
    - Boost your portfolio: Showcase your writing skills and technical knowledge.
```

File: content/en/pages/cfp-program/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: CFP Program
+description: Talk about mirrord
+slug: cfp-program
+header_title: Talk About Mirrord
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:5]:
```suggestion
header_title: Talk About mirrord
```

File: content/en/pages/cfp-program/index.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+header_text: >
+  Are you interested in speaking about mirrord at conferences?
+
+
+  We are calling on mirrored users, contributors, developers, and engineers actively working with mirrord to represent mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:10]:
```suggestion
  We are calling on mirrord users and contributors actively working with mirrord to talk about their takes and experiences with mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.
```

File: content/en/pages/cfp-program/index.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+textbox:
+- label: info
+  title: Program details
+  content: >
+    - We offer financial support to cover the costs of travel, accommodation, and registration fees.
+  
+    - We're especially interested in supporting talks about mirrord, its use cases, or its functionalities.
+    
+    - We will provide personalized assistance to help you deliver smooth presentations
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:19]:
```suggestion
    - We will provide personalized assistance to help you deliver a great presentation.
```

File: content/en/pages/cfp-program/index.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+- label: needs
+  title: What are we looking for?
+  content: >
+    - Subject matter experts: users and individuals with deep knowledge and proven experience with mirrord and its features.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:23]:
```suggestion
    - Subject matter experts: users and individuals with deep knowledge and experience with mirrord.
```

File: content/en/pages/cfp-program/index.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+    - mirrord enthusiasts: Passionate about mirrord's potential and eager to share its benefits with the world.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:25]:
```suggestion
    - mirrord enthusiasts: people who are passionate about mirrord's potential and are eager to share its benefits with the world.
```

File: content/en/pages/cfp-program/index.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+    - Engaging presenters: Ability to deliver captivating presentations that inspire and educate audiences.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:27]:
```suggestion
    - Engaging speakers: speakers who can deliver captivating talks that inspire and educate technical audiences.
```

File: content/en/pages/cfp-program/index.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+    
+    - Individuals from diverse backgrounds and fields are encouraged to apply as long as your topic intersects with the mirrord capabilities.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:29]:
```suggestion
    - Individuals from diverse backgrounds and fields are encouraged to apply as long as your topic intersects with mirrord.
```

File: content/en/pages/cfp-program/index.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+how_to_apply:
+  - "Pick your conferences: Indicate the specific conferences you plan to attend and present at."
+  - "Tell us about your talk: Briefly describe your proposed topic, the target audience, how it aligns with the themes of the conference, and demonstrates mirrord capabilities."
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:32]:
```suggestion
  - "Tell us about your talk: Briefly describe your proposed topic, the target audience, how it aligns with the themes of the conference, and demonstrates mirrord's capabilities and use cases."
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/pages/writers-program/index.md (hunk 3 above)
Comment (Anita-ihuman) [reply content/en/pages/writers-program/index.md:39]: I fixed this

File: content/en/pages/writers-program/index.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+    description: Draft your article based on the agreed-upon outline and style guide. Submit your draft for review and feedback.
+  - title: Revise
+    description: Get expert guidance to elevate your writing! Our editors will offer insightful feedback to help polish your article for publication.
+  - title: Publish
+    description: Once reviews are completed, you will get paid, and we'll publish the article and promote it across all of our channels.
+textbox:
+- label: what?
+  title: What topics should you write about?
+  content: >
+    - Technical tutorials: Using mirrord for different use cases and within different stacks.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:20]:
```suggestion
    - Technical tutorials: using mirrord for different use cases and within different stacks.
```

File: content/en/pages/writers-program/index.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+  
+    - Deep dives: In-depth articles on technical concepts and technologies.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:22]:
```suggestion
    - Deep dives: in-depth articles on technical concepts and technologies.
```

File: content/en/pages/writers-program/index.md [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+    
+    - Best practices: Share tips and tricks for improving developer experience.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:24]:
```suggestion
    - Best practices: share tips and tricks for improving developer experience.
```

File: content/en/pages/writers-program/index.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+    
+    - Personal experiences: Share your own unique journey and lessons learned using mirrord in your software development cycle.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:26]:
```suggestion
    - Personal experiences: share your own unique journey and lessons learned using mirrord in your software development cycle.
```

File: content/en/pages/writers-program/index.md [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+  title: Who can write for us?
+  content: >
+    We welcome applications from everyone, regardless of experience or role. All we ask is that you be comfortable with receiving feedback on your writing.
+
+
+    - We do not publish posts that have already been published somewhere else.
+- label: why?
+  title: Why should you participate?
+  content: >
+    - Get paid: You can earn $150-$300 for each article you write that gets published.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:39]:
```suggestion
    - Get paid: you can earn $150-$300 for each article you write that gets published.
```

File: layouts/_default/writers-program.html [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+                            hope
+                            to provide a vast array of resources that empower developers everywhere to code, run, and
+                            test
+                            software applications smoothly.</p>
+                    </div>
+                    <div class="col-16 col-lg-8">
+                        <p>We work with authors and developers to produce high-quality content on open source, cloud
+                            software development, DevOps, Developer Experience, Platform Engineering, backend
+                            engineering,
+                            and more. All contents are credited to the authors and promoted across our social media,
```
Comment (eyalb181) [inline layouts/_default/writers-program.html:22]:
```suggestion
                            and more. We'll pay you for any content that gets published, and all content is credited to the authors and promoted across our social media,
```

File: content/en/pages/cfp-program/index.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+  content: >
+    - Subject matter experts: users and individuals with deep knowledge and experience with mirrord.
+
+    - mirrord enthusiasts: people who are passionate about mirrord's potential and are eager to share its benefits with the world.
+
+    - Engaging speakers: speakers who can deliver captivating talks that inspire and educate technical audiences.
+    
+    - Individuals from diverse backgrounds and fields are encouraged to apply as long as your topic intersects with mirrord.
+how_to_apply:
+  - "Pick your conferences: Indicate the specific conferences you plan to attend and present at."
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:31]: The first letter after a ":" doesn't need to be capitalized - please fix where relevant.

File: content/en/pages/cfp-program/index.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+textbox:
+- label: info
+  title: Program details
+  content: >
+    - We offer financial support to cover the costs of travel, accommodation, and registration fees.
+  
+    - We're especially interested in supporting talks about mirrord, its use cases, or its functionalities.
+    
+    - We will provide personalized assistance to help you deliver a great presentation.
+- label: needs
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:20]:
```suggestion
- label: requirements
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/pages/cfp-program/index.md (hunk 19 above)
Comment (Anita-ihuman) [reply content/en/pages/cfp-program/index.md]: Fixed

File: content/en/pages/writers-program/index.md [hunk 21]
```diff
@@ -0,0 +1,47 @@
+---
+title: Writers Program
+description: Writers Program
+slug: writers-program
+process:
+  - title: Apply
+    description: submit a brief application with a topic of interest and an outline, along with a writing sample that showcases your experience and writing skills.
```
Comment (eyalb181) [inline content/en/pages/writers-program/index.md:7]: The starts of the descriptions should be capitalized, e.g. Submit

File: content/en/pages/cfp-program/index.md [hunk 22]
```diff
@@ -0,0 +1,36 @@
+---
+title: CFP Program
+description: Talk about mirrord
+slug: cfp-program
+header_title: Talk About mirrord
+header_text: >
+  Are you interested in speaking about mirrord at conferences?
+
+
+  We are calling on mirrord users and contributors actively working with mirrord to talk about their takes and experiences with mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.
```
Comment (eyalb181) [inline content/en/pages/cfp-program/index.md:10]:
```suggestion
  We're inviting mirrord users to talk about their takes and experiences with mirrord at conferences worldwide. Have an idea for a mirrord-related conference talk or workshop? This is your chance to share your knowledge and help spread the word about mirrord.
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**